javascript-stringify.js

```diff
@@ -229,6 +229,11 @@
       return PRIMITIVE_TYPES[typeof value](value, indent, next);
     }
 
+    // Handle Error objects.
+    if (value instanceof Error) {
```

Comment:
@zalmoxisus I can create my own `Error` instances. For instance, in a lot of libraries I'll have more complex errors that may contain sub-errors, etc. All that would be lost because I extended `Error` for stack traces.